pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,249 @@
+              description: "This environment is the live environment available for final users."
+```
+
+### `Tags` in `Channels` object
+Tags are associated with individual channels allowing for logical grouping and categorization of channels based on specific functionalities or business domains. The `tags` object when used within a `channels` object, the context is restricted to the `channels` object and the individual `channel` of the AsyncAPI document, meaning they only impact the `channels` object of the AsyncAPI document. 
+
+Here's a visual representation of the `tags` object inside a `channels` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[channel]
+  B(address)
+  C(title)
+  D(description)
+  E(messages)
+  H(parameters)
+  J(servers)
+  M(bindings)
+  P(tags)
+  R(externalDocs)
+  F(name)
+  G(description)
+  I(externalDocs)
+  Z(asyncapi)
+
+  Z --> A
+  A --> B
+  A --> C
+  A --> D
+  A --> E
+  A --> H
+  A --> J
+  A --> M
+  A --> P
+  A --> R
+  P --> F
+  P --> G 
+  P --> I 
+```
+
+Below is an example of the `tags` object inside the `channels` object in an AsyncAPI document:
+```yaml
+channels:
+  SignedUp:
+    address: 'user.signedup'
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+    tags:
+      - name: user
+        description: User-related messages
+```
+
+### `Tags` in `Operations` Object
+The `tags` object within the `operations` object of the AsyncAPI document allows for logical grouping and categorization of individual `operation` objects based on the type of operation or functionality and more. The `tags` object when used within a `operations` object, the context is restricted to the `operations` object and the individual `operation` of the AsyncAPI document, meaning they only impact the `operations` object of the AsyncAPI document. 
+
+Here's a visual representation of the `tags` object inside a `operations` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[operation]
+  B(title)
+  C(summary)
+  D(description)
+  E(channel)
+  F(action)
+  G(security)
+  H(tags)
+  I(bindings)
+  J(traits)
+  K(messages)
+  L(reply)
+  M(address)
+  N(channel)
+  W(asyncapi)
+  X(name)
+  Y(description)
+  Z(externalDocs)
+  
+  W --> A
+  A --> B
+  A --> C
+  A --> D
+  A --> E
+  A --> F
+  A --> G
+  A --> H
+  A --> I
+  A --> J
+  A --> K
+  A --> L
+  L --> M
+  L --> N
+  H --> X
+  H --> Y
+  H --> Z
+```
+
+Below is an example of the `tags` object inside the `operations` object in an AsyncAPI document:
+```yaml
+operations:
+  onUserSignUp:
+    title: User sign up
+    summary: Action to sign a user up.
+    description: A longer description
+    channel:
+      $ref: '#/channels/userSignup'
+    action: send
+    tags:
+      - name: user
+        description: operation related to user
+      - name: signup
+        description: operation related to a user's signUp
+      - name: register
+        description: operation related to a new registration
```

Comment:
```suggestion
        description: operation related to a new user's registration
```

Reply:
Again, this is just a placeholder text serving as a example, doesn't effect anything.